### PR TITLE
fix a bug, no checking of data_source

### PR DESCRIPTION
**Pull request by sehkone**
When adding new descriptions, the topic name of data_source is now compared as well as its cluster id. Previously, only cluster ids were checked up.

### CODE REVIEW

**Comment by atyoshioka** [comment]
You should update CHANGELOG about the bug fix.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - A new endpoint `GET /api/template` to fetch template(s) from template table.
 - A new endpoint `POST /api/template` to create a new template.
 
+### Fixed
+
+- Check the topic name of clusters whose column statistics are to be created.
```
Comment (atyoshioka) [inline CHANGELOG.md]: Could you briefly describe the issue? You should also describe the change to the endpoint.

Comment (sehkone) [reply CHANGELOG.md]: I did right now.

**Comment by sehkone** [comment]
I'm going to rebase master right away.

**Comment by sehkone** [comment]
> I'm going to rebase master right away.
Do I need to do it? It shows no conflicts with the base. But, you committed a new version right before.

**Comment by sehkone** [comment]
> I'm going to rebase master right away.
> Do I need to do it? It shows no conflicts with the base. But, you committed a new version right before.

Github doesn't update checks as Gitlab does. I did rebase master right now.

**Comment by atyoshioka** [comment]
You didn't have to rebase the branch because there was no conflicts.

**Pull request merged**